packages/api-reference/src/components/GettingStarted.vue

```diff
@@ -2,18 +2,19 @@
 import { ScalarButton } from '@scalar/components'
 import { type ThemeId, themeLabels } from '@scalar/themes'
 
-import placeholderSpecification from '../../../../specifications/scalar-galaxy-3.1.json?raw'
-
 defineProps<{
   theme: ThemeId
 }>()
-
 const emits = defineEmits<{
   (e: 'changeTheme', value: ThemeId): void
   (e: 'loadSwaggerFile'): void
   (e: 'linkSwaggerFile'): void
   (e: 'updateContent', value: string): void
 }>()
+const placeholderSpecificationRes = await fetch(
```

Comment:
fetch the placeholder from the CDN. I was having issues with this relative import